api/v1/observer/observer.proto

```diff
@@ -99,6 +99,10 @@ message GetFlowsRequest {
 
     // Until this time for returned flows. Incompatible with `number`.
     google.protobuf.Timestamp until = 8;
+
+    // first specifies if we should look at the first `number` flows or the
+    // last `number` of flows. Incompatible with `follow`.
+    bool first = 9;
```

Comment:
nit: could we please move this right next to `number` as both are kinda related? On the way, we could also add a note that number == last for better clarification.

Reply:
Pushed up a fix up commit to address this while I work on adding tests. LMK if you think the new wording is good.